app/models/photo.rb

```diff
@@ -1,7 +1,14 @@
 class Photo < ActiveRecord::Base
   belongs_to :project
 
-  validates :image_file_name, presence: true, uniqueness: true
+  validates :image_file_name, presence: true
   validates :image_content_type, presence: true
   validates :image_file_size, presence: true
+
+  has_attached_file :image,
+                    styles: { medium: "300x300", thumb: "100x100" },
+                    default_url: "/images/:styles/beans.png"
+
+  validates_attachment_content_type :image,
+                    content_type: ['image/jpg', 'image/jpeg', 'image/png']
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.